entity-management-web/src/test/java/eu/europeana/entitymanagement/mongo/InsertAndRetrieveEntityToAndFromMongoDB.java

```diff
@@ -0,0 +1,73 @@
+package eu.europeana.entitymanagement.mongo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import eu.europeana.entitymanagement.definitions.model.EntityRecord;
+import eu.europeana.entitymanagement.definitions.model.impl.BaseTimespan;
+import eu.europeana.entitymanagement.definitions.model.mongo.impl.EntityRecordImpl;
+import eu.europeana.entitymanagement.mongo.repository.EntityRecordRepository;
+
+/**
+ * JUnit tests to insert and retrieve the entities from the MongoDB
+ */
+public class InsertAndRetrieveEntityToAndFromMongoDB {
```

Comment:
please rename to EntityRecrodRepositoryTest, the unit test must be named after tested class/interface

Reply:
Renamed.